compiler/modules/types/mapping.bal

```diff
@@ -32,6 +32,13 @@ public class MappingDefinition {
     }
 
     public function define(Env env, Field[] fields, SemType rest) returns SemType {
```

Comment:
I don't think this function should be a method on MappingDefintion. (Idea is that mapping knows nothing about cells.) Rather it should be in core.bal